index.html

```diff
@@ -8,9 +8,9 @@
 
 <link rel="stylesheet" href="style.css">
 
```

Comment:
Kindly update your PR summary to be a bit more descriptive😅. Your summary should summarize all the major changes made in the branch. Something like 

> In this branch, these are the following changes made:
> - Change 1
> - Change 2

Definitely, gives a better idea of what the PR is all about, don't you think?